pkg/domain/podman/common.go

```diff
@@ -18,7 +18,7 @@ const (
 var (
 	Username                = readUsername()
 	SkupperContainerVolumes = []string{"skupper-services", "skupper-local-server", "skupper-internal", "skupper-site-server", SharedTlsCertificates,
-		types.ConsoleServerSecret, types.ConsoleUsersSecret}
+		types.ConsoleServerSecret, types.ConsoleUsersSecret, types.ClaimsServerSecret}
```

Comment:
Is this perhaps a distinct change to adding collector beacons?

Reply:
This is just a reference to the podman volumes that will be created during initialization.
It is also used as a pre-step to the initialization, to ensure skupper is able to create everything it needs beforehand.